Reqnroll.Generator/CodeDom/CodeDomHelper.cs

```diff
@@ -287,6 +287,17 @@ private CodeExpression GetAwaitedMethodThisTargetObject(CodeExpression thisExpre
                 _ => thisExpression
             };
         }
+
+        public string GetGlobalizedTypeName(Type type)
+        {
+            if (TargetLanguage == CodeDomProviderLanguage.CSharp)
+            {
+                return "global::" + type.FullName!;
+            }
+
+            //TODO: VB.NET
```

Comment:
I would not leave this TODO in the code. It is unlikely that we will deal too much with VB. We can keep a comment, saying that we do not support global namespaces for VB, but not a TODO.

Reply:
Agreed. 
Modification made.